<layout & feat> 더보기 창 구현

chaeri93 opened the pull request:
제가 서연벗이 머지하기 전에 이미 작업을 시작해서 테마창은 머지 이후 추후에 작업하도록 하겠습니다. 그리고 그전에 제가 style하고 TouchableOpacity가 적용이 안되는데 혹시 다른분들 작업환경에서 되는지 확인후에 머지하는게 좋을것 같습니다ㅜㅜ
          
![image](https://user-images.githubusercontent.com/79985974/143826505-9d8d37f5-ff07-42c5-bd35-9add6abb0b05.png)
```
<View style={viewStyles.menuView}>
                <TouchableOpacity onPress={() => {setDeleteMode(!DeleteMode), setModalVisible(false)}}/>
                <Text style={textStyles.moremenu}> Delete </Text>
            </View>

            <View style={viewStyles.horizentalline}></View>
```

이부분이 잘 안되네요ㅜㅜ 혹시 제 코드에 틀린부분이 있으시면 말씀해주세요! 그리고 드래그앤 드롭하고 이외에 체크박스도 다시 넣으려고 했는데 둘다 모듈 설치가 필요하더라구요,,,,

flowersayo commented:
Delete버튼을 눌렀을때 삭제모드로 들어가는걸 구현하고 싶으신거죠?! 

1. `setModalVisible` 은 그 투두 세부사항창을 띄우는지 여부(bool)를 설정하는 setter 함수여서 저 코드는 버튼이 눌리면 
세부사항창을 보이지마라하는 의미인것 같아요. 관련없는 부분이라 아예 삭제하셔도 될것 같아요!

2.  지금 필요한건 더보기창 띄울지 말지를 결정하는 bool형  state 변수랑, 삭제모드여부를 판별하는 bool형 state변수를 생성해서 onPress에 넣어주는것 같아요! 원래 제 PR까지는 삭제관련 state변수를 받아서 topbar의 내용을 바꿔주는 부분이 존재했는데 현아PR로 가면서 사라진것같아요ㅠㅠ. 

3.  문장과 문장 사이는 `;` 로 이어야할것 같아요!  저는 화살표함수때문에 처음에 엄청 이해하는데 헤맸었는데 
저는 이거 참고하고 좀 괜찮아졌어요 ....>,,<  

https://poiemaweb.com/es6-arrow-function

4.  `<TouchableOpacity>` 는 여는 태그랑 닫는태그 둘다 있어야하는것같아요! 그래야 그 안에 내용물을 touch 가능하도록 할 수 있으니까요 

----

```
  const [ExtraVisible, setExtraVisible] = useState(false); // 더보기창을 보이고 있는지 여부
  const [DeleteMode, setDeleteMode] = useState(false); //삭제모드인지 여부 


<View >
    <TouchableOpacity onPress={() => {setDeleteMode(!DeleteMode), setExtraVisible(false)}}>
    <Text style={textStyles.moremenu}> Delete </Text>
    </TouchableOpacity>
</View>

...
``` 

아마 이런식에다가  현아벗 PR에서 빠진 이부분 추가해주시면 될 것 같아요

```

if (DeleteMode){ //삭제모드라면 -> 상단바부분을 삭제부분으로 변경.
    TopBar=<View style={viewStyles.settingView} >
    <IconButton type={images.back}  onPressOut={() => setDeleteMode(!DeleteMode)}/>
    <Text>  Delete </Text>
    <View style={viewStyles.settingGroup}>
      <IconButton type={images.unchecked} />
      <IconButton type={images.trash} />
    </View>
  </View>

  }
```

flowersayo committed:
initial setting

flowersayo committed:
[<refactor> 메인홈 간소화 및 모달창 연결 구현

`...` 버튼과 더보기창 연결 , `>` 버튼과 세부모달창 연결, 앱아이콘 추가 및 아이콘 버튼과 theme선택창 연결

Yevin-WIN committed:
<style> 세부사항모달창 분리 및 코드 정리

1. DetailTodolist.js에 세부사항모달창분리
2. todo-Icon 이미지 제대로 나오도록 변경
3. Memo.js 글자 크기 맞춤
4. 그 외 자잘한 코드 정리

Yevin-WIN committed:
<style> 세부사항모달창 분리 및 코드 정리

DetailTodolist.js에 세부사항 모달창(todolist >버튼 누르면 나오는 창) 분리

Yevin-WIN committed:
<style> 세부사항모달창 분리 및 코드 정리

DetailTodolist.js에 세부사항 모달창(todolist >버튼 누르면 나오는 창) 분리

Yevin-WIN committed:
<feat> 이미지 업로드 기능 추가

세부모달창에 expo-image-picker 모듈 이용하여 이미지 업로드 기능 추가

Yevin-WIN committed:
<chore> 모듈 관련 수정사항

"react-navigation": "^4.4.4",
"expo-image-picker": "~11.0.3",

chaeri93 committed:
Merge pull request #1 from Open-SW-platform/Yevin-WIN

<style,feat,chore> 세부사항모달창분리, 이미지업로드기능, 모듈추가

chaeri93 committed:
<fix> 모듈 문제 해결
